tools/osbuild-mpp

```diff
@@ -642,6 +649,11 @@ LocalFileSigLevel = Optional
                 pkg["name"],
                 pkg["version"],
                 pkgdata["Architecture"],
+                pkg.get("Licenses"),
```

Comment:
Does this really work? pkg comes from the output of `--print-format ...` which hasn't changed. Maybe these are supposed to come from pkgdata?

Reply:
Good question, just in case I have disabled auto merge on this PR temporary.

Reply:
Sorry, indeed yes

Reply:
Adjusted